doc/using-logging.rst

```diff
@@ -0,0 +1,117 @@
+=====================
+ Configuring Logging
+=====================
+
+One common use of ZConfig is to configure the Python :mod:`logging`
+framework. ZConfig provides one simple convenience function to do this:
+
+.. autofunction:: ZConfig.configureLoggers
+
+
+This function takes the configuration text::
+
+    >>> from ZConfig import configureLoggers
+    >>> configureLoggers('''
```

Comment:
That's fine. 

Just understand that conversion of doctests to run under manuel is trivial. :)

Something like::

            manuel.testing.TestSuite(
                manuel.doctest.Manuel(
                    optionflags=doctest.NORMALIZE_WHITESPACE | doctest.ELLIPSIS
                    ) + manuel.capture.Manuel(),
                'foo.txt',
                setUp=docSetUp, tearDown=setupstack.tearDown
                ))

Instead of:

            doctest.DocFileSuite(
                'foo.txt',
                setUp=docSetUp, tearDown=setupstack.tearDown
                optionflags=doctest.NORMALIZE_WHITESPACE | doctest.ELLIPSIS
                )

I did this for several projects when we got rid of the zope.testing.doctest fork.